CHANGELOG.md

```diff
@@ -15,3 +15,8 @@ To learn about API versioning, see
 ## March 2023
 
 * Launched Webhooks API.
+
+## September 2023
+
+* Partner authentication is now supported see [FAQ](https://developer.vippsmobilepay.com/docs/APIs/webhooks-api/faq/).
```

Comment:
Nice to add a note to the changelog! I think you can drop the link because it implies that we've written something specific about partner authentication in the faq, but I couldn't find anything... Do we need something?

Reply:
Maybe we can link to your new section in the API guide though.